docs/specification/2024-11-05/basic/messages.mdx

```diff
@@ -27,8 +27,7 @@ Requests are sent from the client to the server or vice versa.
 
 - Requests **MUST** include a string or integer ID.
 - Unlike base JSON-RPC, the ID **MUST NOT** be `null`.
-- The request ID **MUST NOT** have been previously used by the requestor within the same
```

Comment:
while this overlaps with the JSON-RPC spec a bit, the old wording is more correct -- the ID is not unique in the session, but only for the requestor. The client and server can make requests using the same ID.